pyntc/devices/eos_device.py

```diff
@@ -193,6 +220,11 @@ def enable(self):
 
     @property
     def uptime(self):
```

Comment:
This property assigns only and does not explicitly return. I think it would be better to omit the `Returns` statement in the docstring.